StreamChatUITestsAppUITests/Tests/ChannelList_Tests.swift

```diff
@@ -6,6 +6,53 @@ import XCTest
 
 final class ChannelList_Tests: StreamTestCase {
 
+    let message = "message"
+
+    func test_newMessageShownInChannelPreview_whenComingBackFromOffline() {
```

Comment:
Hm, but there's no connectivity stuff happening the scenario, right?

Reply:
@evsaev good catch. I just updated the function name ;)